changelog/_unreleased/2025-07-07-use-minimal-search-term-length-in-config-tables.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Use minimal search term length in config tables
+issue: 8018
+---
+# Core
```

Comment:
please add t he upgrade information that what this change affect the configuration